content/sensu-go/5.16/guides/monitor-sensu.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "Monitor Sensu with Sensu"
+linkTitle: "Monitor Sensu with Sensu"
+description: "Make sure your Sensu components are properly monitored. This guide describes best practices and strategies for monitoring Sensu."
+weight: 230
+version: "5.16"
+product: "Sensu Go"
+platformContent: False
+menu: 
+  sensu-go-5.16:
+    parent: guides
+---
+
+
+- [Monitor your Sensu backend instances](#monitor-your-sensu-backend-instances)
```

Comment:
Checks that could potentially comprise a fully-operational battlestation:
* Service checks for `sensu-backend`
* Port checks for the backend component ports
* Test queries against the api
* Prom exporter check against the /metrics endpoint

Reply:
Some additional checks here depending on if using embedded etcd or not. If using external etcd, there would be port/service(maybe) checks that would need to be relocated. 

Also, what about federation? We're in the practice of recommending a gateway be stood up--we'd need to figure out what would need to be monitored for a gateway that isn't processing events. Moreover, we might want a check that looks to see if a gateway somehow starts processing events, as that would indicate something is misconfigured

Reply:
I am not sure exactly how to incorporate this content. I did add the table you suggested -- maybe that is a good jumping-off point?

Reply:
Agent websocket API check similar to backend `health` endpoint check. Need to check if there is one and if there isn't, ask for one through a FR against Sensu Go.

Reply:
Moving to action item in document.